packages/block-library/src/cover/edit.native.js

```diff
@@ -142,10 +149,12 @@ const Cover = ( {
 	const isImage = backgroundType === MEDIA_TYPE_IMAGE;
 
 	const THEME_COLORS_COUNT = 4;
-	const colorsDefault = useSetting( 'color.palette' ) || [];
-	const coverDefaultPalette = {
-		colors: colorsDefault.slice( 0, THEME_COLORS_COUNT ),
-	};
+	const colorsDefault = useMobileGlobalStylesColors();
+	const coverDefaultPalette = useMemo( () => {
+		return {
+			colors: colorsDefault.reverse().slice( 0, THEME_COLORS_COUNT ),
+		};
+	}, [ colorsDefault ] );
```

Comment:
I realized that the issue "Standard themes - Default colors are reverse ordered" described in [this review](https://github.com/WordPress/gutenberg/pull/38417#pullrequestreview-871753314), is partially caused due to using `reverse` here. `reverse` modifies the original array so on every render it's reversing the colors for the entire editor.

We could make a copy by doing the following:
```suggestion
	const coverDefaultPalette = useMemo( () => {
		return {
			colors: colorsDefault.slice().reverse().slice( 0, THEME_COLORS_COUNT ),
		};
	}, [ colorsDefault ] );
```

Reply:
Oh nice suggestion! With the latest changes, it doesn't use reverse anymore. I'll explain why the order was showing up differently between platforms in a comment below.